examples/java/build.gradle

```diff
@@ -91,10 +91,18 @@ dependencies {
   implementation "org.apache.commons:commons-lang3:3.9"
   implementation "org.apache.httpcomponents:httpclient:4.5.13"
   implementation "org.apache.httpcomponents:httpcore:4.4.13"
+  implementation project(path: ":runners:flink:1.11")
```

Comment:
I added that one so that I could use the FlinkPipelineOptions to set the setFasterCopy to true. When running the example through flink it recommends setting that to true. Is it ok to leave it there? Not sure about the pros and cons.